### ISSUE TITLE
Monsters glitch trough blocks when hit

### ISSUE THREAD

**Issue by 99thGamer9** [cuberite/cuberite#5279]
Client version: 1.12.2
Server OS: Linux
Cuberite Commit id: 6a4460383e98fbdbdf568c0cb154dafec833ed44

### Expected behavior
Mob gets hit against but not through the wall

### Actual behavior
Mob gets glitches trough the wall

### Steps to reproduce the behavior
Hit a monster that stands in front of a wall, works best when the wall is made of blocks like slime or iron bars

**Comment by Ccode-lang** [comment]
So see through blocks?

**Comment by tigerw** [comment]
Bounding box issue, see http://youtu.be/EA-w8HZ9_Fg (#898).

PR #4776 should fix it but it's a little stuck right now.

**Comment by 99thGamer9** [comment]
> So see through blocks?

No, if you hit them (or push them with a piston) they can glitch trough semi-solid blocks
